ixofoundation/ixo-webclient
# feat: add project filter buttons

## Scope
As a user, I can filter by certain categories including All Projects and My Projects. 

## Work Done
Add filter dropdown with buttons to filter

## Steps to Test
Run the app and make sure the buttons are interact-able and look like the designs

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/74738912-7adf1600-5260-11ea-8e72-9dbbef95aab2.png)
